tests/test_tools.py

```diff
@@ -0,0 +1,13 @@
+from functools import partial
+
+from rasterio.features import dataset_features
+from rasterio.tools import JSONSequenceTool
+
+
+def test_dataset_features(tmpdir):
+    """JSON sequence tool works with dataset_features"""
+    tool = JSONSequenceTool(partial(dataset_features, bidx=1, as_mask=True))
```

Comment:
You could say I'm partial to this function from functools.

This one test gives us 100% coverage for the new tools module.